src/main/java/net/bestemor/villagermarket/menu/EditItemMenu.java

```diff
@@ -233,6 +235,11 @@ private void typeAmount(Player player) {
             }
             player.sendMessage(ConfigManager.getMessage("messages.amount_successful"));
             shopItem.setAmount(result.intValue());
+            EditShopItemEvent editShopItemEvent = new EditShopItemEvent(player,this.shop, shopItem);
+            Bukkit.getPluginManager().callEvent(editShopItemEvent);
+            if (editShopItemEvent.isCancelled()) {
+                return;
+            }
```

Comment:
Should be moved before the `sendMessage` and `setAmount`

Reply:
If you place it before the setAmount function, you won't get the updated shop item. Should I keep it as it is or modify the event parameter, like (player, shop, shopItem, newValue) ?

Reply:
I think modifying the event parameter would be best. Otherwise, cancelling it would have no effect